charts/humio-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: humio-operator
-version: 0.28.2
-appVersion: 0.28.2
+version: 0.28.3
+appVersion: 0.28.3
```

Comment:
Shouldn't we do this at release time? The generated charts have `helm.sh/chart: 'humio-operator-0.28.2'`

Reply:
Oh. Right, I missed this part. Updating now. Correct. Let's keep it at 0.28.2 for now